avocado-setup.py

```diff
@@ -266,6 +267,18 @@ def install_optional_plugin(plugin):
         # plugin already installed
         pass
 
+def configure_prepostscript(repos, scripttype):
+    i=0
+    cmd= 'mkdir -p /etc/avocado/scripts/job/%s.d' % scripttype
+    helper.runcmd(cmd, ignore_status=True, info_str="%s script directory created" % scripttype)
+    for repo in repos:
```

Comment:
why do we have a loop for every repo?

Reply:
ah, repos you mean scripts, repos in the wrapper denotes different meaning, no doc string